### PR TITLE
Add Feature Release ticket template

### PR DESCRIPTION
**Pull request by mrjones-plip**
# Description

Add Feature Release ticket template

 per #7609

# Code review checklist

- [X] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [X] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by mrjones-plip** [comment]
@garethbowen - do we need to change the version in `package.json` and `package-lock.json`? I wanted to be sure what ever showed up in the hamburger menu and telemetry docs is accurate!

@craig-landry  - tried to keep it as slim as possible - open to suggestions! 

Both - Wanted this to be stand alone and not make any assumptions about what future folks will know/assume, as well as have it make sense in the context of general public reading this.

**Review by craig-landry — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/z_release_feature.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] Create a GH Milestone for the release.
+- [ ] Add all the issues to be worked on to the Milestone.
+- [ ] Have an actual named deployment and specific end user that will be testing this Feature Release. They need to test in production, on the latest version. No speculative Feature Releases.
+- [ ] Assign an engineer as Release Engineer for this release.
+
+# Development
+
+- [ ] Create a new release branch in `cht-core` from the most recent release and call it  `<major>.<minor>.<patch>-FR-<FEATURE-NAME>`. If latest is `3.15.0` and the feature is to "allow movies to be uploaded", call it `3.15.0-FR-movie-upload`. Done before the release so all PRs can be set to merge to this branch, and not to `master`.
+- [ ] Set the version number in `package.json` and `package-lock.json` and submit a PR. The easiest way to do this is to use `npm --no-git-tag-version version <feature-release>`.
+- [ ] Ensure QA is debriefed and is partnering with the Trio to ensure early and often checks of the feature are on track to be of production quality from the start.
```
Comment (garethbowen) [inline .github/ISSUE_TEMPLATE/z_release_feature.md:23]:
```suggestion
- [ ] Ensure QA is briefed and is partnering with the Trio to ensure early and often checks of the feature are on track to be of production quality from the start.
```

**Comment by garethbowen** [comment]
> do we need to change the version in package.json and package-lock.json? I wanted to be sure what ever showed up in the hamburger menu and telemetry docs is accurate!

Technically both. Using the recommended approach (as you've documented in the template) then it updates both automatically.

**Review by garethbowen — APPROVED** [review]
Great. One minor improvement inline.

**Pull request merged**